submission/distributor.go

```diff
@@ -30,8 +30,22 @@ import (
 	"github.com/google/certificate-transparency-go/x509"
 
 	ct "github.com/google/certificate-transparency-go"
+	"github.com/google/trillian/monitoring"
```

Comment:
nit: move this up into the block of un-aliased imports

Reply:
Done